src/pages/ReportSettingsPage.js

```diff
@@ -0,0 +1,211 @@
+                            {this.props.translate('newRoomPage.roomName')}
+                        </Text>
+                        <View style={[styles.flexRow]}>
+                            <View style={[styles.flex3]}>
+                                <RoomNameInput
+                                    onChangeText={newRoomName => this.setState({newRoomName})}
+                                    onChangeError={error => this.setState({error})}
+                                    initialValue={this.state.newRoomName}
+                                    disabled={shouldDisableRename}
+                                    policyID={linkedWorkspace && linkedWorkspace.id}
+                                />
+                            </View>
+                            <Button
+                                success={!shouldDisableRename}
+                                text={this.props.translate('common.save')}
+                                onPress={() => {
+                                    // When renaming is built, this will use that API command
+                                }}
+                                style={[styles.ml2]}
+                                textStyles={[styles.label]}
+                                buttonStyles={[styles.reportSettingsChangeNameButton]}
+                                isDisabled={Boolean(
+                                    shouldDisableRename
+                                    || this.state.newRoomName === this.props.report.reportName
+                                    || this.state.error,
+                                )}
+                            />
+                        </View>
+                    </View>
+                    {linkedWorkspace && (
+                        <View style={[styles.mt4]}>
+                            <Text style={[styles.formLabel]} numberOfLines={1}>
+                                {this.props.translate('workspace.common.workspace')}
+                            </Text>
+                            <Text numberOfLines={1}>
+                                {linkedWorkspace.name}
+                            </Text>
+                        </View>
+                    )}
+                    {this.props.report.visibility && (
+                        <View style={[styles.mt4]}>
+                            <Text style={[styles.formLabel]} numberOfLines={1}>
+                                {this.props.translate('newRoomPage.visibility')}
+                            </Text>
+                            <Text numberOfLines={1}>
+                                {/* Use capialized versions of the visibility rNVP value */}
+                                {
+                                    this.props.report.visibility === CONST.REPORT.VISIBILITY.RESTRICTED
+                                        ? 'Restricted'
+                                        : 'Private'
+                                }
```

Comment:
Can we just use built in JS functions to capitalize here to avoid hardcoding stuff?
```suggestion
    {/* Use capitalized versions of the visibility rNVP value */}
    {`${this.props.report.visibility[0].toUpperCase()}${this.props.report.visibility.slice(1)}`}
```